src/obu.rs

```diff
@@ -122,6 +122,29 @@ impl Debug {
     }
 }
 
+fn check_trailing_bits(gb: &mut GetBits, strict_std_compliance: bool) -> Rav1dResult {
+    // trailing_one_bit + trailing_zero_bit
+    if !gb.get_bit() || gb.pending_bits() != 0 || gb.has_error() != 0 {
+        return Err(EINVAL);
+    }
+
+    if !strict_std_compliance {
+        return Ok(());
+    }
+
+    let mut size = gb.remaining_len();
+
+    while size > 0 && gb[size - 1] == 0 {
+        size -= 1;
+    }
+
+    if size != 0 {
+        return Err(EINVAL);
+    }
```

Comment:
```suggestion
    if !gb.get_bytes(gb.remaining_len()).iter().all(|&b| b == 0) {
        return Err(EINVAL);
    }
```

Reply:
Thank you for the suggestion. Had to add a call to `gb.bytealign()` to avoid an assertion failure in `gb.get_bytes()`